mad-dog/src/index.js

```diff
@@ -29,6 +30,27 @@ const {
   allUp
 } = ServiceCommands
 
+const numContentNodes = 3
```

Comment:
how will this work if there are 4 content nodes? in fact I think there are by default if you scroll up in this file and check out `DEFAULT_NUM_CREATOR_NODES`

Reply:
if we specify 4 content nodes, it will fail the health check because there is no actual cn4 up

Reply:
disregard my last comment, the default number of CNs is 4. will update